ts/components/FiscalCodeLandscapeOverlay.tsx

```diff
@@ -69,7 +88,13 @@ export default class FiscalCodeLandscapeOverlay extends React.PureComponent<
           backgroundColor={customVariables.brandDarkGray}
           barStyle={"light-content"}
         />
-        <Content style={styles.content}>
+        <ScrollView
+          style={styles.content}
+          ref={this.ScrollVewRef}
+          onContentSizeChange={(_, height) => {
+            this.scrollViewHeight = height; // tslint:disable-line no-object-mutation
```

Comment:
You should use **state** instead of class field
Anyway you can remove this _onContentSizeChange_, check the comment about usage of **_scrollToEnd_**